about.html

```diff
@@ -0,0 +1,97 @@
+                    <li><a href="#" target="_blank">logout</a></li>
+                </ul>
+            </nav>
+        </div>
+        <nav class="navbar">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <a href="#" class="logo">GTH.</a>
+
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="index.html" active class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#program" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#join" class="nav-link">Featured</a></li>
+                <li class="nav-item"><a href="#sponser" class="nav-link">Partners</a></li>
+                <li class="nav-item"><a href="#index.html" class="live nav-link">Register</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="about-section">
+        <div class="title about-title">
+            <h1>GLOBAL-TECH <br> HACKATHON <br> 2022</h1>
+        </div>
+        <div class="description about-desc">
+            <p>
+                The GT hackerthon aims to bring in startups, developers, strategists, and thinkers to explore Digital
+                opportunities and help solve pressing problems faced by large corporates and startups in key industry
+                sectors.Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
+                has been the industry's standard dummy text ever since the 1500s.
+        </div>
+        <p class="our-email">Please contact us per Email for any further questions about globaltech-hackathon 2022</p>
+
+        <a class="email" href="#">globaltech-hackathon@gmail.com></a>
+    </section>
+
+    <section class="content">
+        <h2 class="top-speakers about-sp"> Global-Tech Hackathon 2022 Logo</h2>
+        <div class="line"></div>
+        <p class="about-text"> Global-Tech Hackathon 2022 logo was designed by Preccious Nhlapho in 2022.</p>
+        <img class="about-logo" src="./pictures/siteLogo.png" alt="GtH Logo">
+
+    </section>
+    <div class="hr-line"></div>
+    <section class="content">
+        <h2 class="top-speakers about-sp">See the past Global-Tech Hackathons</h2>
+        <div class="line"></div>
+        <p class="about-text">Take a look at the past winners of GLOBAL-TECH Hackathon 2021</p>
+
+        <div class="winners-images">
+            <img class="winners-image" src="./pictures/winners2.jpg" alt="Winner1">
+            <img class="winners-image" src="./pictures/winners1.jpg" alt="Winner2">
+        </div>
+    </section>
+
+
+    <footer>
+        <div class="footer abt-footer">
+            <div>
+                <p class="copyright"><span>&copy</span>Global-Tech Hakathon 2022 <br>
+                    By Precious Nhlapho</p>
+            </div>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] it would be better if the footer on the about page should be white in the mobile version in order to match the design 

- [ ] [OPTIONAL] Also you should put the logo on the right side of the footer with the text in the right as in the Design in order to match the design 

- Yours 
<img width="402" alt="Screen Shot 2022-10-28 at 15 11 59" src="https://user-images.githubusercontent.com/61790587/198637933-a2440952-c646-4a1a-8312-e8127f68c7bb.png">
- Design 
<img width="482" alt="Screen Shot 2022-10-28 at 15 15 55" src="https://user-images.githubusercontent.com/61790587/198638495-076f86c5-2160-40cb-bd26-aff103160e76.png">